### PR TITLE
Restaurantpage

### PR DESCRIPTION
**Pull request by samgaco**
Submitting for a code review

### CODE REVIEW

**Review by JamesOkunlade — CHANGES_REQUESTED** [review]
# Changes required 🙂 

**Good job so far @samgaco . Here are some points to get you a thumb up on this project:**

- [x] The `about`, `contact`, and `menu` files in your `src` folder do not have file name extension. a `.js` should be appended to each of these files.
- [x] All necessarily used packages should be included in `package.json` like `webpack`, `webpack-cli`, `webpack-dev-server` etc. Also, those packages should be added as devDependencies
- [x] Necessary run common commands like `build`, `development` should be in `package.json` inside `scripts`. Here is a basic example
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```

**Fix these points and request for re-review when you're done.**

**James Okunlade**
**Happy coding! 😎**

**Comment by samgaco** [comment]
Added all the suggested changes.

File: src/about.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  const render = () => {
+    document.getElementById('content').innerHTML = '';
+    addtag('#content', 'H1', 'レストラン');
+    addimg('#content', './images/header.png');
+    addtag('#content', 'P', 'Our food is salty, sweet, sour, slightly bitter and full of umami flavors, and equal emphasis is placed on our beautiful presentations.');
+    addtag('#content', 'P', '私たちの食べ物は、塩辛く、甘く、酸っぱく、わずかに苦く、うま味がいっぱいです。美しいプレゼンテーションにも同様に重点が置かれています。');
+  };
+
+  return { render };
+})();
```
Comment (BalintMendli) [inline src/about.js:31]: Since we are now using ES6 modules there's no need for this IIFE module pattern. This file is a module in itself.

**Review by BalintMendli — APPROVED** [review]
Hi @samgaco ,

The restaurant page looks great. 👏 

Great job implementing the requested changes.

### 👍 

You can go ahead and merge the project.

**Pull request merged**